tests/test_connection.py

```diff
@@ -44,3 +43,14 @@ def test_Misc(self):
         result[0].uri['nodelist'].pop()
         self.assertTrue(len(result[0].uri['nodelist']) == 0)
         self.assertEqual("Connection()", repr(self.named_conn))
+
+    @defer.inlineCallbacks
+    def test_uri_input(self):
+        test = txmongo.connection.ConnectionPool()
+        yield test.disconnect()
+        test = txmongo.connection.ConnectionPool("mongodb://127.0.0.1/dbname")
+        yield test.disconnect()
+        test = txmongo.connection.ConnectionPool(u"mongodb://127.0.0.1/dbname")
```

Comment:
That's OK, we are trying to find problems between python2 and python3 with regard to the default string behaviour. 

Adding a b"blah" in python2 will work just fine, but in python3 it will error out an AssertionError.